templates/vue/src/components/Lightbox/media/ActivityMedia/ListQuestion.vue

```diff
@@ -0,0 +1,173 @@
+<template>
+  <div class="container">
+    <div class="list">
+      <ul
+        v-for="(answer, index) in answerList"
+        :key="answer.index"
+        class="answerItem"
+      >
+        <b-form-input
+          v-model="answerList[index]"
+          :data-qa="`list-input-${index}`"
+          :placeholder="
+            question.answerTypes.list.placeholder
+              ? question.answerTypes.list.placeholder
+              : 'Enter text and press Enter'
+          "
+        ></b-form-input>
+        <b-button
+          :disabled="numOfFields >= maxFields"
+          variant="primary"
+          :data-qa="`list-add-${index}`"
+          :class="{
+            'btn-primary': !(numOfFields >= maxFields),
+            disabled: numOfFields >= maxFields,
+          }"
+          @click="addAnswer"
+          @keydown.enter.prevent="addAnswer"
+        >
+          +
+        </b-button>
+        <b-button
+          :disabled="numOfFields <= minFields"
+          variant="danger"
+          :data-qa="`list-del-${index}`"
+          :class="{
+            'btn-danger': !(numOfFields <= minFields),
+            disabled: numOfFields <= minFields,
+          }"
+          @click="deleteAnswer(index)"
+        >
+          -
+        </b-button>
+      </ul>
+    </div>
+    <div class="submission">
+      <b-button
+        class="submit-btn"
+        data-qa="list-submit-btn"
+        variant="primary"
+        @click="handleListSubmit"
+      >
+        Submit
+      </b-button>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: "list-question",
+  props: {
+    question: {
+      type: Object,
+      required: true,
+    },
+    answers: {
+      type: Array,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      answerList: this.answers.map(x => x),

```

Comment:
I think I had it as `answerList: this.answers` before but that makes answerList a reference to the answers prop, which I didn't want because it messes up the submission (answers will be saved even if user doesn't click submit). The identity map makes a new array that resolves this.